Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/lucene/impl/LuceneUtility.java

```diff
@@ -622,15 +622,19 @@ private void buildQuery(BooleanClause.Occur occur, PerFieldAnalyzerWrapper perFi
 
     private void buildTokenizedOrWildCardQuery(LuceneField luceneFieldAnnotation, String searchString,
             BooleanQuery.Builder tokenizedQueryBuilder) {
-        for (String searchValue : searchString.split(" ")) {
+        String hypSearchStr = new String(searchString);
+        hypSearchStr = searchString.replace('-', ' ');
             if (luceneFieldAnnotation.isWildCardSearchEnabled()) {
+                for (String searchValue : hypSearchStr.split(" ")) {
                 createWildCardSearchQuery(luceneFieldAnnotation, searchValue, tokenizedQueryBuilder, Occur.MUST);
+                }
             } else {
+                for (String searchValue : hypSearchStr.split(" ")) {
                 tokenizedQueryBuilder.add(new PhraseQuery(luceneFieldAnnotation.lucenefieldName(), searchValue),
                         Occur.MUST);
+                }
```

Comment:
this is not the solution for the problem; we will have the same problem most likely if there are other word separating characters (e.g. :, ~, etc). You need to make sure the search terms are normalized/analyzed before the query is created.

Reply:
This change was merged to develop as part of Normalized value search change